models/app.models.js

```diff
@@ -1,8 +1,16 @@
 const db = require("../db/connection");
+const fs = require("fs/promises");
 
 const notFoundErrorObj = { status: 404, msg: "not found" };
 const badRequestErrorObj = { status: 400, msg: "bad request" };
 
+exports.readEndpoints = () => {
```

Comment:
This is great, but you can move it to the controller. 

The model is reserved for functions which access the database. There is no need for a model in this endpoint.